static/js/timerender.js

```diff
@@ -164,6 +164,52 @@ exports.render_date = function (time, time_above, today) {
     return node;
 };
 
+// Renders the timestamp returned by the !time() markdown syntax.
+exports.render_markdown_timestamp = function (time, now, text) {
+    now = now || moment();
+    let timestring = time.format('ddd, MMM D');
+    if (now.year() !== time.year()) {
+        timestring += time.format(' YYYY');
+    }
+    timestring += ',';
```

Comment:
This kind of code is going to be tricky to localize. Moment has a fairly comprehensive collection of auto-localizing formatting codes, can we just use those?

Reply:
Yeah, I'd love to use something from `moment` rather than this big block of code if there's something appropriate.